var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -34,20 +34,26 @@ class Crtm(CMakePackage):
     depends_on("netcdf-fortran", when="@v2.3-jedi.4")
     depends_on("netcdf-fortran", when="@v2.4-jedi.1")
     depends_on("netcdf-fortran", when="@v2.4-jedi.2")
+    depends_on("netcdf-fortran", when="@v2.4.1-jedi")
 
     depends_on("crtm-fix@2.3.0_emc", when="@2.3.0 +fix")
     depends_on("crtm-fix@2.4.0_emc", when="@2.4.0 +fix")
 
     depends_on("ecbuild", type=("build"), when="@v2.3-jedi.4")
     depends_on("ecbuild", type=("build"), when="@v2.4-jedi.1")
     depends_on("ecbuild", type=("build"), when="@v2.4-jedi.2")
+    depends_on("ecbuild", type=("build"), when="@v2.4.1-jedi")
 
     # ecbuild release v2.4.0 is broken
     # add ecbuild dependency for next release with fix
     # depends_on("ecbuild", when="@2.4.0:", type=("build"))
 
     license("CC0-1.0")
 
+    version(
+        "v2.4.1-jedi", sha256="fd8bf4db4f2a3b420b4186de84483ba2a36660519dffcb1e0ff14bfe8c6f6a14"
+    )
```

Comment:
I can't seem to confirm this version sha256 with `spack checksum`.  How was it calculated?